challenges/intermediate-generic/solution.py

```diff
@@ -0,0 +1,19 @@
+"""
+TODO:
+
+The function `add` accepts two parameters of the same type and returns the same type.
+"""
+
+from typing import TypeVar
+
+T = TypeVar('T')
+def add(a, b: T) -> T:
+    return a
+
```

Comment:
ditto, add this too
```
def add[T](a, b: T) -> T:
    return a
```

Reply:
By design `a` should also be `T`, if it passes the test then the test doesn't cover.

Reply:
Sg, we can add the test case first

Reply:
@nobod4t I updated the test case (https://github.com/laike9m/Python-Type-Challenges/commit/795aa6abcaa0cd7d39f6f79959cbe70ffe9176e0), could you pull the latest code and update the solution as suggested?